src/textual/css/_styles_builder.py

```diff
@@ -181,10 +221,20 @@ def process_box_sizing(self, name: str, tokens: list[Token]) -> None:
                     self.error(
                         name,
                         token,
-                        f"invalid value for box-sizing (received {value!r}, expected {friendly_list(VALID_BOX_SIZING)})",
+                        string_enum_help_text(
+                            "box-sizing",
+                            valid_values=list(VALID_BOX_SIZING),
```

Comment:
Isn't `VALID_BOX_SIZING` a set? Sets have undefined order. Are we sorting the valid values? The should probably be alphabetical.

Reply:
Most of these are ultimately sorted inside `friendly_list`, but there are one or two places they don't get sorted. I'll add that.